Makefile.toml

```diff
@@ -22,6 +22,7 @@ skip_crate_env_info = true
 
 [env]
 CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+RUSTFLAGS = "-D warnings"
```

Comment:
Using `-D warnings` for `cargo make` while not using the same `RUSTFLAGS` for regular build invalidates the building cache.

You can run `./ci_checks.sh` twice a row, and you will see that it starts building from scratch.